src/Core/Utilities/StaticStore.cs

```diff
@@ -500,8 +129,8 @@ static StaticStore()
                     GetPlan(org.PlanType).Product == ProductType.Enterprise,
             }
         };
-    public static Plan GetPlan(PlanType planType) =>
-        Plans.FirstOrDefault(p => p.Type == planType);
+    public static Plan GetPlan(PlanType planType, BitwardenProductType bitwardenProduct = BitwardenProductType.PasswordManager) =>
+        Plans.FirstOrDefault(p => p.Type == planType && p.BitwardenProduct == bitwardenProduct);
```

Comment:
Might be best to use `SingleOrDefault` or we could end up hiding a plan without knowing it. Also add a unit test to test this so we're not adding two plans with the same `PlanType` and `BitwardenProductType` to the `PlansStore` because `SingleOrDefault` throws an exception if there's more than one result.